pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+u"""run black formatter on file/dir
+
+:copyright: Copyright (c) 2013-2018 Bivio Software, Inc.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+
+
+def edit(path):
+    cmd = ['black', f'{path}']
+    pykern.pksubprocess.check_call_with_signals(cmd)
+
+
+def diff(path, path_expect=None):
```

Comment:
Not sure what path_expect means. I think expect means what we expect it to be, that is the reference value. I don't see why there would need to be a reference value passed to this function.